docs/get-started/quickstarts/collect-data.md

```diff
@@ -0,0 +1,209 @@
+---
+title: "Collect images or sensor data in 2 minutes"
+linkTitle: "Collect data in 2 minutes"
+type: "docs"
+weight: 20
+cost: 75
+images: ["/platform/data.svg"]
+description: "Use Viam to gather images or sensor data from your machine."
+---
+
+Follow this guide to start collecting images or sensor data from your machine using the [data management service](/services/data/):
+
+## Requirements
+
+- A Linux or macOS computer:
+  - A single-board computer (SBC) such as a Raspberry Pi, OR
+  - A laptop or desktop
+- A camera or a sensor.
+  Options:
```

Comment:
I think my original "Options" feels sorta short and blunt, but I also am not sure about "Options include" because I think these are all of the options, not just some of them, so I think I'll leave as-is for now